src/Cameras/Inputs/BaseCameraPointersInput.ts

```diff
@@ -0,0 +1,328 @@
+import { Nullable } from "../../types";
```

Comment:
Interesting but need to be cleaned if we want to use it as base class for all pointer inputs

Reply:
@deltakosh I presume your just busy but want to make sure your aware i've cleaned this up.
I'm not in any hurry at my end so whenever works for me.

cheers!
dunk.